seed/seed20.go

```diff
@@ -416,20 +472,10 @@ func (s *seed20) LoadMeta(tm timings.Measurer) error {
 
 			// done with essential snaps
 			essential = false
-		}
-	}
-
-	// extra snaps
-	runMode := []string{"run"}
-	for {
-		optSnap, done := s.nextOptSnap(nil)
-		if done {
-			break
-		}
-
-		_, err := s.addSnap(optSnap, optSnap, runMode, "latest/stable", "snaps", tm)
-		if err != nil {
-			return err
+			if essentialOnly {
+				// will not find more
```

Comment:
Is this because gadget is the last essential snap? Can you extend the comment?

Reply:
yes,  it's not clear in the diff maybe, but the line before is exactly

essential = false